cmd/spire-server/cli/bundle/common.go

```diff
@@ -17,6 +19,15 @@ import (
 	"github.com/zeebo/errs"
 )
 
+const (
+	headerFmt = `****************************************
+* %s
+****************************************
+`
+	formatPEM  = "pem"
+	formatJWKS = "jwks"
```

Comment:
What do you think about `"spiffe"` for this format instead of JWKS?

Reply:
works for me!